### PR TITLE
Added seekable start and end for Live dvr to work in Flashls 

### PR DESCRIPTION
**Pull request by shahlabs**
Live-Dvr in flashls doesn't work, it shows --/-- . To fix that , we need to add seekable-start and seekable-end as vjs_properties and use them in flashls-source-handler.

Requires the following for full support:
https://github.com/brightcove/videojs-flashls-source-handler/pull/1

### CODE REVIEW

File: src/com/videojs/providers/IProvider.as [hunk 1]
```diff
@@ -26,6 +26,16 @@ package com.videojs.providers{
         function get duration():Number;
 
         /**
+         * Should return a value that indicates the current seekable start, in seconds.
+         */
+        function get seekStart():Number;
```
Comment (OshinKaramian) [inline src/com/videojs/providers/IProvider.as]: I always appreciate how interface classes make folks add changes to 3 extra files :)

**Review by OshinKaramian — APPROVED** [review]

**Comment by OshinKaramian** [comment]
Before merging this up we should probably merge the source handler changes / version / link it here.

File: src/VideoJS.as [hunk 2]
```diff
@@ -250,6 +250,12 @@ package{
                 case "duration":
                     return _app.model.duration;
                     break;
+                case "seekStart":
```
Comment (mjneil) [inline src/VideoJS.as]: I think we should name these `seekableStart` and `seekableEnd` to avoid any confusion with seek actions

Comment (mjneil) [reply src/VideoJS.as]: It also might be good to add a comment here, (or within the other providers) mentioning that these values are only meant to be used with the HLSProvider

File: src/com/videojs/providers/HTTPAudioProvider.as [hunk 3]
```diff
@@ -134,6 +134,18 @@ package com.videojs.providers{
             return [];
         }
 
+        /** HTTPAudioProvider does not support seekableStart
```
Comment (mjneil) [inline src/com/videojs/providers/HTTPAudioProvider.as]: sorry for the nit-picky, but can you fix the formatting on these comments (also in the other providers)
```
/**
 * HTTPAudioProvider does not support seekableStart
 */
```

**Comment by mjneil** [comment]
I don't think it matters too much which we merge/version first, but getting everything into the swf and versions first makes sense to me since the source handler currently would work the same with the old or new swf, but the new source handler after merge will only work with the new swf

edit: since we are also going to be adding the quality-levels integration, we might as well wait until that is also completed before versioning

**Pull request merged**